virttest/test_setup.py

```diff
@@ -399,9 +399,12 @@ def get_target_hugepages(self):
             hugepage_allocated = open(self.kernel_hp_file, "r")
             available_hugepages = int(hugepage_allocated.read().strip())
             hugepage_allocated.close()
-            chunk_bottom = int(math.log(self.hugepage_size / 4, 2))
-            chunk_info = utils_memory.get_buddy_info(">=%s" % chunk_bottom,
-                                                     zones="DMA32 Normal")
+            chunk_bottom = int(math.log(self.hugepage_size / (os.sysconf('SC_PAGE_SIZE')/1024), 2))
```

Comment:
Suggest introducing a `getpagesize()` function to `virttest/arch.py` and call it here. BTW, the `resource` module provides such the function.

Reply:
@luckyh Sure, I feel this place can be placed, `virttest/staging/utils_memory.py` as we are directly reading from `SC_PAGE_SIZE`

Reply:
OK, make sense.